tools/topology/topology1/m4/dai.m4

```diff
@@ -155,7 +155,7 @@ define(`DO_DAI_CONFIG',
 `'
 `	id		"'$3`"'
 `'
-`	ifelse($1, `SSP', $5, $1, `HDA', $5, $1, `ALH', $5, $1, `ESAI', $5, $1, `SAI', $5, $1, `AFE', $5, $1, `ACP', $5, $1, `ACPSP', $5, $1, `ACPDMIC', $5, `}')'
+`	ifelse($1, `SSP', $5, $1, `HDA', $5, $1, `ALH', $5, $1, `ESAI', $5, $1, `SAI', $5, $1, `AFE', $5, $1, `ACP', $5, $1, `ACPSP', $5, $1, `ACPHS', $5, $1, `ACPDMIC', $5, `}')'
```

Comment:
Fwiw, topology2 will fix this sort of expanding macros.

Reply:
> Fwiw, topology2 will fix this sort of expanding macros.

@lgirdwood :Can you plz merge the pull request as all approval's came for this pull request.